### ISSUE TITLE
[BUG] Cannot start CRC on latest ubuntu 20.10

### ISSUE THREAD

**Issue by dgolovin** [code-ready/crc#1623]
### General information

  * OS: Linux 
  * Hypervisor: KVM
  * Did you run `crc setup` before starting it (Yes)?
  * Running CRC on: Laptop 64Gb RAM

## CRC version
```bash
CodeReady Containers version: 1.17.0+99f5c87
OpenShift version: 4.5.14 (embedded in binary)
```
  
## CRC status
```bash
CRC VM:          Stopped
OpenShift:       Stopped
Disk Usage:      0B of 0B (Inside the CRC VM)
Cache Usage:     38.56GB
Cache Directory: /home/denis/.crc/cache
```

## CRC config
```bash
no output
```

## Host Operating System
```bash
NAME="Ubuntu"
VERSION="20.10 (Groovy Gorilla)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.10"
VERSION_ID="20.10"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=groovy
UBUNTU_CODENAME=groovy
```

### Steps to reproduce

  1. src setup
  2. src start 

### Expected

crc started

### Actual

fails with

```
$./crc start -p ../pull-secret.txt 
INFO Checking if oc binary is cached              
INFO Checking if podman remote binary is cached   
INFO Checking if goodhosts binary is cached       
INFO Checking minimum RAM requirements            
INFO Checking if running as non-root              
INFO Checking if Virtualization is enabled        
INFO Checking if KVM is enabled                   
INFO Checking if libvirt is installed             
INFO Checking if user is part of libvirt group    
INFO Checking if libvirt daemon is running        
INFO Checking if a supported libvirt version is installed 
INFO Checking if crc-driver-libvirt is installed  
INFO Checking if libvirt 'crc' network is available 
INFO Checking if libvirt 'crc' network is active  
INFO Checking if NetworkManager is installed      
INFO Checking if NetworkManager service is running 
INFO Checking if /etc/NetworkManager/conf.d/crc-nm-dnsmasq.conf exists 
INFO Checking if /etc/NetworkManager/dnsmasq.d/crc.conf exists 
INFO Loading bundle: crc_libvirt_4.5.14.crcbundle ... 
INFO Checking size of the disk image /home/denis/.crc/cache/crc_libvirt_4.5.14/crc.qcow2 ... 
INFO Creating CodeReady Containers VM for OpenShift 4.5.14... 
ERRO Error creating machine: Error creating the VM: Error creating machine: Error in driver during machine creation: virError(Code=1, Domain=10, Message='internal error: process exited while connecting to monitor: 2020-10-29T07:18:12.992977Z qemu-system-x86_64: -blockdev {"driver":"file","filename":"/home/denis/.crc/cache/crc_libvirt_4.5.14/crc.qcow2","aio":"threads","node-name":"libvirt-2-storage","auto-read-only":true,"discard":"unmap"}: Could not open '/home/denis/.crc/cache/crc_libvirt_4.5.14/crc.qcow2': Permission denied') 
Error creating machine: Error creating the VM: Error creating machine: Error in driver during machine creation: virError(Code=1, Domain=10, Message='internal error: process exited while connecting to monitor: 2020-10-29T07:18:12.992977Z qemu-system-x86_64: -blockdev {"driver":"file","filename":"/home/denis/.crc/cache/crc_libvirt_4.5.14/crc.qcow2","aio":"threads","node-name":"libvirt-2-storage","auto-read-only":true,"discard":"unmap"}: Could not open '/home/denis/.crc/cache/crc_libvirt_4.5.14/crc.qcow2': Permission denied')

$ls -la /home/denis/.crc/cache/crc_libvirt_4.5.14/crc.qcow2
-rwxrwxrwx 1 libvirt-qemu kvm 10709827584 Oct 28 23:55 /home/denis/.crc/cache/crc_libvirt_4.5.14/crc.qcow2

```

### Logs

see above

**Comment by dgolovin** [comment]
crc 1.16.0 works fine.

**Comment by cfergeau** [comment]
This is a duplicate of https://github.com/code-ready/crc/issues/1578 which has a workaround. I don't think it's been fixed yet in git though, so maybe #1578 should be reopened

**Comment by dgolovin** [comment]
Workaround gets me to

```
INFO Checking size of the disk image /home/denis/.crc/cache/crc_libvirt_4.5.14/crc.qcow2 ... 
INFO Creating CodeReady Containers VM for OpenShift 4.5.14... 
ERRO Error creating machine: Error creating the VM: Error creating machine: Error in driver during machine creation: virError(Code=1, Domain=10, Message='internal error: Process exited prior to exec: libvirt:  error : internal error: cannot apply process capabilities -1') 
Error creating machine: Error creating the VM: Error creating machine: Error in driver during machine creation: virError(Code=1, Domain=10, Message='internal error: Process exited prior to exec: libvirt:  error : internal error: cannot apply process capabilities -1')
```

Any workaround for this one?

**Comment by dgolovin** [comment]
@cfergeau NM, I found the problem. I've made a mistake in AppArmor config. 
The workaround works for me. Thanks!

**Comment by dgolovin** [comment]
crc 1.18.0 still has the same problem.
